src/main/java/FRIDAY/FRIDAYException.java

```diff
@@ -0,0 +1,7 @@
+package FRIDAY;
+
+public class FRIDAYException extends Exception {
```

Comment:
Perhaps renaming the class to `FridayException` would be more compliant with the coding standards